docs/production-deployment/self-hosted-guide/checklist.mdx

```diff
@@ -23,9 +23,39 @@ tags:
   - versioning
 ---
 
-Consider the following when running a production level Temporal Service.
+This page is intended to describe the most common challenges for customers who choose to self-host Temporal and share recommendations to mitigate those challenges. 
 
-## Scaling and Metrics {#scaling-and-metrics}
+Temporal at its core is about durability and reliability.
+This durability is compromised if it is not architected and deployed as a reliable, trusted platform.
+This guide provides a path to demonstrate that Temporal consumers can be confident in a Temporal service and provides a list of key tests you as a user should perform against the service, such as:
+
+This guide provides a path to demonstrate that Temporal consumers can be confident in a Temporal service and provides a list of key tests you as a user should perform against the service.
+
+## Self-Hosting Challenge Areas
+
+Significant engineering must be done and ongoing effort is required to resolve these challenges:
+- Scalability with spiky or growing workloads
+- Global hosting
+- Uptime, availability and reliability
+- Management and control plane
+- Latency, which must be kept low and consistent
+- [Security](/self-hosted-guide/security)
+- Maintenance and upgrades
+- Expert support to users of the service
+- Cost management
+
+It is critical to demonstrate confidence in these areas as part of building a mission critical Temporal Service.
+Without demonstrated architectural durability, the value of Temporal's [Durable Execution](https://temporal.io/how-it-works) model is compromised.
+
+## Scalability with Variable or Growing Workloads {#scaling-and-metrics}
+
+Workloads can be highly variable, and sustained workload spikes of 5-10x average is common. 
+We recommend scaling your clusters to well above the average throughput.
```

Comment:
I'd love to know if and when we can use "We". I know Marketing had some thoughts about this. Generally I ended up changing "We" to "Temporal" if I wasn't sure.

Reply:
By the way, the advice is excellent.

Reply:
*Some* of our legacy docs have "we," and because these are recommendations, it's hard to remove it outright. Anyone we should @ here for a head check?

Reply:
Actually, removed after second round of feedback.